### PR TITLE
Add close_widget to hud_battle_log

### LINKED ISSUE #1728: View -> show combat detailed information doesn't seem to do anything
**Describe the bug**
On the View menu is an option called "show combat detailed information". It does not seem to toggle anything.

**To Reproduce**
1. Open a game
2. Select an attacking unit
3. Middle-Click on an enemy unit on a tile to see the pop up widget
4. Notice the combat information shown
5. Toggle the menu item off
6. Do steps 2 to 4 again
7. Notice the same information is shown.

**Expected behavior**
Menu options do something useful

**Screenshots**
n/a

**Platform and version (please complete the following information):**
 - OS: Linux
 - Freeciv21 version: master
 - Ruleset/Longturn game (if applicable): n/a

**Additional context**
None

### PR DESCRIPTION
**Pull request by jwrober**
Initial stab at adding a close_widget "x" to the battle results widget. Some things that still need figuring out

- [x] Determine why the widget is "expanded" to the whole right hand side of the pop up widget
- [x] Determine how to shift the scale buttons to the left 12 pixels so they won't overlap the "x" when the first item is figured out.

This is what it looks like right now
image

- [x] Add client manual documentation

Closes #1728

### CODE REVIEW

**Comment by lmoureaux** [comment]
The two dots you can see at the top right are a `move_widget`. The `close_widget` at the right location, but much larger than usual:
image
(Used `gammaray` to see this.)

File: docs/Manuals/client-manual.rst [hunk 1]
```diff
@@ -543,7 +543,25 @@ Show New Turn Information
     Enables or Disables populating new turn information in a widget on the `Map View`_.
 
 Show Detailed Combat Information
-    Enables or Disables populating the middle-click pop-up widget with combat information.
+    Enables or Disables populating the :guilabel:`Battle Log` widget. When enabled you will see a widget
+    appear on the screen (typically in the upper left corner) after conflict occurs in your nation's vision.
```
Comment (lmoureaux) [inline docs/Manuals/client-manual.rst:547]:
```suggestion
    appear on the screen (typically in the upper left corner) when a combat occurs in your nation's vision.
```

**Review by lmoureaux — COMMENTED** [review]
Small suggestion for the docs, otherwise looks good. This may cross the "feature" line a bit too much for a backport, but still ok to add it.

**Comment by jwrober** [comment]
Good point on the feature push. Maybe I should do a client manual update without a screen shot for stable as it is there.

**Pull request merged**